### PR TITLE
Add recipe for pyscrap

### PR DESCRIPTION
**Pull request by npavlovikj**
Add recipe for pyscrap

### CODE REVIEW

File: recipes/pyscrap/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+package:
+  name: pyscrap
+  version: '0.0.9'
+
+source:
+  fn: pyscrap-0.0.9.tar.gz
+  url: https://pypi.python.org/packages/aa/cb/07ac450abb5c2eb7cdf52ceb394c8a35e4cca440e59e964a029d0696f181/pyscrap-0.0.9.tar.gz
```
Comment (CJ-Wright) [inline recipes/pyscrap/meta.yaml]: Please use jinja templating for the url. See the example.

File: recipes/pyscrap/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+  run:
+    - python
+    - simplejson
+    - lxml
+    - libxml2
+    - libxslt
+
+test:
+  imports:
```
Comment (CJ-Wright) [inline recipes/pyscrap/meta.yaml]: Would it be possible to add more import tests?

File: recipes/pyscrap/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+      - pyscrap
+
+about:
+  home: https://github.com/Zincr0/pyscrap
+  license: Apache-2.0
```
Comment (CJ-Wright) [inline recipes/pyscrap/meta.yaml]: This needs a license file and a license family

Comment (npavlovikj) [reply recipes/pyscrap/meta.yaml]: The license file is only in the repo, and not part of tar.gz.

File: recipes/pyscrap/meta.yaml (hunk 1 above)
Comment (npavlovikj) [reply recipes/pyscrap/meta.yaml]: Thanks for the feedback @CJ-Wright , I appreciate it! I believe I fixed the things you suggested, but the build still fails. Can you help me with this when you have a chance please?

File: recipes/pyscrap/meta.yaml (hunk 3 above)
Comment (CJ-Wright) [reply recipes/pyscrap/meta.yaml]: https://conda-forge.org/docs/meta.html#packaging-the-license-manually

**Comment by CJ-Wright** [comment]
I'm not certain what is going on with this, it seems that circle is failing at the beginning of the build: `/bin/bash: ./scripts/run_docker_build.sh: No such file or directory`
@conda-forge/core thoughts?

**Comment by isuruf** [comment]
Please merge with master

File: recipes/pyscrap/meta.yaml (hunk 3 above)
Comment (npavlovikj) [reply recipes/pyscrap/meta.yaml]: Ah, thank you @CJ-Wright !

**Pull request closed without merging**